src/app/auth/auth-routing.module.ts

```diff
@@ -1,8 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
```

Comment:
you need to redirect the user if he went directly to `/auth`  to the `/auth/login`